<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>517</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>6</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>输配协同下面向薄弱点关联潮流精准调控的多元用户联动决策支持系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#151a1e;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>15</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>5</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="font">
          <font>
           <pointsize>1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QStackedWidget{	
	border-width: 1px;
	border-style: solid;
	border-color: #4fa08b;
	
	color: #d3dae3;
}

background-color: #222b2e;</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFrame" name="frame3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">

.QFrame{	
	border-width: 1px;
	border-style: solid;
	border-color: #4fa08b;
	background-color: #222b2e;
	color: #d3dae3;
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: #222b2e;
font: 13pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>新能源消纳薄弱点及对应的关键线路编号及其对应潮流</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>7</horstretch>
                  <verstretch>9</verstretch>
                 </sizepolicy>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QFrame" name="frame_8">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>1</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                        <horstretch>0</horstretch>
                        <verstretch>1</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: #222b2e;
font: 13pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
                      </property>
                      <property name="text">
                       <string>新能源消纳率可视化图</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="m1_p1">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                        <horstretch>0</horstretch>
                        <verstretch>6</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: #222b2e;
font: 13pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_9">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>1</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                        <horstretch>0</horstretch>
                        <verstretch>1</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: #222b2e;
font: 13pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
                      </property>
                      <property name="text">
                       <string>线路传输容量可视化图</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="m1_p2">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                        <horstretch>0</horstretch>
                        <verstretch>6</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: #222b2e;
font: 13pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m1_run1">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>3</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
	border-style: solid;
	border-color: rgb(90, 180, 180);
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #151a1e;
}
QPushButton::default{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #FFFFFF;
	padding: 2px;
	background-color: #151a1e;;
}
QPushButton:hover{
	border-style: solid;
	border-color: rgb(24, 255, 140);
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #1c1f1f;
}
QPushButton:pressed{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #2c2f2f;
}</string>
                </property>
                <property name="text">
                 <string>运行</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">
.QFrame{	
	border-width: 1px;
	border-style: solid;
	border-color: #4fa08b;
	background-color: #222b2e;
	color: #d3dae3;
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: #222b2e;
font: 13pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>新能源消纳薄弱点对应的可调柔性负荷节点编号排序</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>9</verstretch>
                 </sizepolicy>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QFrame" name="frame_10">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>1</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: #222b2e;
font: 13pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
                      </property>
                      <property name="text">
                       <string>可调柔性负荷节点编号排序</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="m1_p3">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                        <horstretch>0</horstretch>
                        <verstretch>6</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color: #222b2e;
font: 13pt &quot;楷体&quot;;
color: rgb(255, 255, 255);</string>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m1_run2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
	border-style: solid;
	border-color: rgb(90, 180, 180);
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #151a1e;
}
QPushButton::default{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #FFFFFF;
	padding: 2px;
	background-color: #151a1e;;
}
QPushButton:hover{
	border-style: solid;
	border-color: rgb(24, 255, 140);
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #1c1f1f;
}
QPushButton:pressed{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #2c2f2f;
}</string>
                </property>
                <property name="text">
                 <string>运行</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page2">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QFrame" name="frame_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QFrame" name="frame_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>2</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m2_btn1">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>3</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
	border-style: solid;
	border-color: rgb(90, 180, 180);
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #151a1e;
}
QPushButton::default{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #FFFFFF;
	padding: 2px;
	background-color: #151a1e;;
}
QPushButton:hover{
	border-style: solid;
	border-color: rgb(24, 255, 140);
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #1c1f1f;
}
QPushButton:pressed{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #2c2f2f;
}</string>
                </property>
                <property name="text">
                 <string>导入电网数据</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_4">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>2</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m2_btn2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>3</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
	border-style: solid;
	border-color: rgb(90, 180, 180);
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #151a1e;
}
QPushButton::default{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #FFFFFF;
	padding: 2px;
	background-color: #151a1e;;
}
QPushButton:hover{
	border-style: solid;
	border-color: rgb(24, 255, 140);
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #1c1f1f;
}
QPushButton:pressed{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #2c2f2f;
}</string>
                </property>
                <property name="text">
                 <string>导入运行数据</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_6">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>2</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m2_btn3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>3</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
	border-style: solid;
	border-color: rgb(90, 180, 180);
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #151a1e;
}
QPushButton::default{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #FFFFFF;
	padding: 2px;
	background-color: #151a1e;;
}
QPushButton:hover{
	border-style: solid;
	border-color: rgb(24, 255, 140);
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #1c1f1f;
}
QPushButton:pressed{
	border-style: solid;
	border-color: #050a0e;
	border-width: 1px;
	border-radius: 5px;
	color: #d3dae3;
	padding: 2px;
	background-color: #2c2f2f;
}</string>
                </property>
                <property name="text">
                 <string>运行评估</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_7">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>2</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="m2_p1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>9</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>22</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">	border-width: 1px;
	border-style: solid;
	border-color: #4fa08b;
	background-color: #222b2e;
	color: #d3dae3;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>11</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
	background-color: #151a1e;
 	border: 1px solid rgb(90, 180, 180); /* 菜单栏的边框 */;
}
QMenuBar::item {
	color: #d3dae3;
  	spacing: 3px;
  	padding: 1px 4px;
	background-color: #151a1e;
}

QMenuBar::item:selected {
  	background-color: #252a2e;
	color: #FFFFFF;
}
QMenu {
	background-color: #151a1e;
}
QMenu::item:selected {
	background-color: #252a2e;
	color: #FFFFFF;
}
QMenu::item {
	color: #d3dae3;
	background-color: #151a1e;
}</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>功能模块</string>
    </property>
    <addaction name="action1"/>
    <addaction name="action2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="to_module1">
   <property name="text">
    <string>模块一</string>
   </property>
  </action>
  <action name="to_module2">
   <property name="text">
    <string>模块二</string>
   </property>
  </action>
  <action name="to_module3">
   <property name="text">
    <string>模块三</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>模块一</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>模块二</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
